### DISCUSSION TITLE
Read from the cache on page load/page refresh.

### DISCUSSION THREAD

**Discussion by davidhan527** [TanStack/query#545] [General]
Hello,

I tried reading through the documentation/issues and I couldn't find an answer to this question.

Is there any way to read from the cache on the initial page load or when you refresh the browser page, or will it always be in a loading state during every page refresh?

**Comment by kumarwa** [comment]
@tannerlinsley I have similar question, and womndering if you could provide an answer for it

Is there any way to read from the cache on the initial page load or when you refresh the browser page, or will it always be in a loading state during every page refresh?
